nav2_msgs/srv/GetCostmap.srv

```diff
@@ -4,3 +4,5 @@
 nav2_msgs/CostmapMetaData specs
 ---
 nav2_msgs/Costmap map
+geometry_msgs/PoseStamped pose
```

Comment:
I don't think you are using the PoseStamped msg in this srv in the code and you are already creating a GetRobotPose srv, so I think these additional fields should be removed.

Reply:
Good catch. Thanks.